dotCMS/src/main/java/com/dotmarketing/util/ConvertToPOJOUtil.java

```diff
@@ -175,46 +114,24 @@ private static boolean isFieldPresent(final Class classToUse, Class fieldType, S
     }
 
     @NotNull
-    private static Link getLinkFields(Map<String, String> map) throws ParseException {
+    private static Link getLinkFields(Map<String, Object> map) throws ParseException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.text.ParseException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)